scripts/paypalBillingSetup.js

```diff
@@ -73,7 +73,7 @@ switch(OP) {
     });
     break;
   case "create":
-    paypal.billingPlan.create(blocks["google_6mo"].definition, function(err,plan){
+    paypal.billingPlan.create(blocks["focusme_6mo"].definition, function(err,plan){
```

Comment:
Quickly confirming that yes, we want to keep the preexisting discount.

On Sat, Jun 4, 2016 at 9:43 AM, Matteo Pagliazzi <notifications@github.com>
wrote:

> In scripts/paypalBillingSetup.js
> <https://github.com/HabitRPG/habitrpg/pull/7598#discussion_r65804695>:
>
> > @@ -73,7 +73,7 @@ switch(OP) {
> >      });
> >      break;
> >    case "create":
> > -    paypal.billingPlan.create(blocks["google_6mo"].definition, function(err,plan){
> > +    paypal.billingPlan.create(blocks["focusme_6mo"].definition, function(err,plan){
>
> @SabreCat <https://github.com/SabreCat> sounds good!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7598/files/5d06ba0964521fdeb9d76e16cf659da4ca112253#r65804695>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADVQwXeUWRHfSrzLMEhQ4z-MhGIjttb0ks5qIatNgaJpZM4It6oN>
> .
>